articles/fleetctl.md

```diff
@@ -20,6 +20,16 @@ sudo npm install -g fleetctl
 
 To install fleetctl on Windows or Linux, download the fleectl binary here on [GitHub](https://github.com/fleetdm/fleet/releases).
 
+### Additional dependencies
+
+To use `fleetctl package` for platforms other than Windows, you'll need to run `fleetctl` from macOS or Linux.
+
+To use `fleetctl package` to create macOS installers (.pkg files), if you're using Linux you'll need Docker, as we use utilities in the `fleetdm/bomutils` Docker image there.
+
+To use `fleetctl package` to create Windows installers (.msi files), you'll need Docker if you aren't on an Apple Silicon Mac. This is because we use the `fleetdm/wix` Docker image to provide the [WiX](https://github.com/wixtoolset/wix3) dependency on most platforms. For Apple Silicon Macs, we automatically install WiX locally when needed.
```

Comment:
Wine is also required (and not automatically installed) on Apple silicon:
https://github.com/fleetdm/fleet/blob/9da53572767df989367919a3249f4b3c65b697a5/orbit/pkg/packaging/windows.go#L177-L181

Nit, maybe worth splitting dependencies into two bullets:
- macOS Intel: ...
- macOS Apple Silicon: ...